docs/lesson/lesson-schema.yaml

```diff
@@ -0,0 +1,19 @@
+definitions:
+  lessonBody:
+    type: object
+    properties:
+      title:
+        type: string
+        minLength: 1
+        maxLength: 100
+      description:
+        type: string
+        minLength: 1
+        maxLength: 1000
+      attachments:
+        type: array
+        items:
+          type: string
+    required:
+      - title
+      - description
```

Comment:
you forgot to add "author" to required list

Reply:
author will be added inside the controller, it's not necessary to put it to the body